ci/build_cpp.sh

```diff
@@ -11,6 +11,7 @@ rapids-print-env
 
 rapids-logger "Begin cpp build"
 
-rapids-mamba-retry mambabuild conda/recipes/libcudf
+rapids-mamba-retry mambabuild \
+    conda/recipes/libcudf
 
```

Comment:
Suggesting a slight tweak to make it easier to add more lines here in the future

```suggestion
    conda/recipes/libcudf \


```

Reply:
Sorry, what is this for? I don't understand. You can't end a command with an escape sequence. I don't know what new lines would be needed.

Reply:
No strong feelings here. Feel free to make resolved if desired